### DISCUSSION TITLE
Localhost:3000 says Project with name “<Name of iModel>” does not exist

### DISCUSSION THREAD

**Discussion by richhum3** [iTwin/itwinjs-core#355] [Q&A] [answered]
I have the simple viewer app running on localhost. Once I have clicked on 'Open Sample iModel', I get message 'localhost:3000 says Project with name "<Name of iModel" does not exist'. I believe its me not setting imjs_test_imodel and imjs_test_project correctly. I've tried a number of times trying different syntax and then rebuilding the app, only to get the same stumbling block. No doubt its me but any advice greatly appreciated. Extract of the lines I have edited within .env.local below:

imjs_test_imodel="via-model"

imjs_test_project="via-model"

**Comment by shenshengyi** [comment, marked as answer]
Hi, simpleviewer app program I have registered and used many times, never had a problem.

So, please check:

1. Is via-model registered successfully? And received an email, https://www.itwinjs.org/getting-started/registration-dashboard?tab=1

2. https://connect-imodelhubwebsite.bentley.com/ Can I open your registered via-model on this website

3. If 1 and 2 are OK, it may be yours local.env The configuration file has not been modified. It is recommended to debug the obtained imjs_test_What is imodel? Is it a registered via-model
